presto-kafka/src/test/java/io/prestosql/plugin/kafka/util/TestingKafka.java

```diff
@@ -48,12 +48,14 @@ public void close()
         container.close();
     }
 
-    public void createTopics(String topic)
+    public void createTopics(String... topics)
```

Comment:
if we do not need functionality of creating multiple topics at once, it'd be better to 
rename the method to `createTopic(String)`

Reply:
+1

Reply:
changed